feature/gnoi/factory_reset/tests/factory_reset_test/README.md

```diff
@@ -0,0 +1,18 @@
+# FACTORY RESET 
```

Comment:
Please fix the following error:

Error reading testdir: feature/gnoi/factory_reset/tests/factory_reset_test: could not parse README.md: cannot parse markdown: # FACTORY RESET 

The format must be regexp.MustCompile(`#(.*?):(.*)`), e.g., `# XX-1.1: Foo Functional Test`

Reply:
This test is gNOI-6.1 Factory Reset